vitals/Message.java

```diff
@@ -0,0 +1,82 @@
+package extensions;
+
+public class Message {
+	enum TEMPERATURE
+	{
+		low_breach("LOW_TEMPERATURE_BREACH"),
+		high_breach("HIGH_TEMPERATURE_BREACH"),
+		low_warning("LOW_TEMPERATURE_WARNING"),
+		high_waring("HIGH_TEMPERATURE_WARNING"),
+		normal("NORMAL");
+		
+		private String message;
+		
+		TEMPERATURE(String message) {
+			this.message = message;
+		}
+
+		public String getMessage() {
+			return message;
+		}
+	}
+	
+	enum SOC
+	{
+		low_breach("LOW_SOC_BREACH"),
+		high_breach("HIGH_SOC_BREACH"),
+		low_warning("LOW_SOC_WARNING"),
+		high_waring("HIGH_SOC_WARNING"),
+		normal("NORMAL");
+
+		private String message;
+		
+		SOC(String message) {
+			this.message = message;
+		}
+
+		public String getMessage() {
+			return message;
+		}
+	}
+	
+	enum CHARGERATE
+	{
+		breach("CHARGERATE_BREACH"),
+		warning("CHARGERATE_WARNING"),
+		normal("NORMAL");
+
+		private String message;
+		
+		CHARGERATE(String message) {
+			this.message = message;
+		}
+
+		public String getMessage() {
+			return message;
+		}
+	}
+	
+	static void printMessage(String type, String category) {
+		
+		switch(category)
+		{
+		case "Temperature":

```

Comment:
println duplicated ....why don't reuse printMessage(string) method 

printMessage(string type,string category)  open for modification and closed for extensibility ...check whether u can refactor this code using "map" datastructure or statergy pattern